pkg/fqdn/name_manager_gc.go

```diff
@@ -23,9 +23,7 @@ const DNSGCJobInterval = 1 * time.Minute
 
 const dnsGCJobName = "dns-garbage-collector-job"
 
-var (
-	dnsGCControllerGroup = controller.NewGroup(dnsGCJobName)
-)
+var dnsGCControllerGroup = controller.NewGroup(dnsGCJobName)
```

Comment:
Please refrain from this style-only changes, they make it more difficult to focus on the core changes of the patch (during review and after the merge too, in case of a bisect) and to backport it to previous stable versions.
If the current version is ok for the active linters and you don't need to change the code, it's better to leave it as is.

Reply:
Agree, any stylistic changes should be paired with a linter that ensures everyone writes code the same way. The current Go standard linters/formatters don't reformat these lines, so I don't think it's a good idea to shuffle these lines around.